lib/smooth/include/smooth/application/io/i2c/AxpRegisters.h

```diff
@@ -0,0 +1,142 @@
+        Reg8BH_Vbus_Detect_Srp = 0x8B,
+        Reg8FH_Ovr_Temp_Shutdn = 0x8F,
+
+        // Group 2 - GPIO Control
+        Reg90H_Gpio0_Func_Set = 0x90,
+        Reg91H_Gpio0_Volt_Set = 0x91,
+        Reg92H_Gpio1_Func_Set = 0x92,
+        Reg93H_Gpio2_Func_Set = 0x93,
+        Reg94H_Gpio_012_Status = 0x94,
+        Reg95H_Gpio_34_Fnc_Set = 0x95,
+        Reg96H_Gpio_34_Status = 0x96,
+        Reg97H_Gpio_012_PD_Ctl = 0x97,
+        Reg98H_Pwm1_Freq_Set = 0x98,
+        Reg99H_Pwm1_Duty_Set1 = 0x99,
+        Reg9AH_Pwm1_Duty_Set2 = 0x9A,
+        Reg9BH_Pwm2_Freq_Set = 0x9B,
+        Reg9CH_Pwm2_Duty_Set1 = 0x9C,
+        Reg9DH_Pwm2_Duty_Set2 = 0x9D,
+        Reg9EH_NRSTO_Func_Set = 0x9E,
+
+        // Group 3 - Interrupt Control
+        Reg40H_Irq_Enable_Grp1 = 0x40,
+        Reg41H_Irq_Enable_Grp2 = 0x41,
+        Reg42H_Irq_Enable_Grp3 = 0x42,
+        Reg43H_Irq_Enable_Grp4 = 0x43,
+        Reg4AH_Irq_Enable_Grp5 = 0x4A,
+        Reg44H_Irq_Status_Grp1 = 0x44,
+        Reg45H_Irq_Status_Grp2 = 0x45,
+        Reg46H_Irq_Status_Grp3 = 0x46,
+        Reg47H_Irq_Status_Grp4 = 0x47,
+        Reg4DH_Irq_Status_Grp5 = 0x4D,
+
+        // Group 4 - ADC Values
+        Reg56H_Acin_Volt_HI8 = 0x56,
+        Reg57H_Acin_Volt_LO4 = 0x57,
+        Reg58H_Acin_Curr_HI8 = 0x58,
+        Reg59H_Acin_Curr_LO4 = 0x59,
+        Reg5AH_Vbus_Volt_HI8 = 0x5A,
+        Reg5BH_Vbus_Volt_LO4 = 0x5B,
+        Reg5CH_Vbus_Curr_HI8 = 0x5C,
+        Reg5DH_Vbus_Curr_LO4 = 0x5D,
+        Reg5EH_Intr_Temp_HI8 = 0x5E,
+        Reg5FH_Intr_Temp_LO4 = 0x5F,
+        Reg62H_Batt_Temp_HI8 = 0x62,
+        Reg63H_Batt_Temp_LO4 = 0x63,
+        Reg64H_Gpio0_Volt_HI8 = 0x64,
+        Reg65H_Gpio0_Volt_LO4 = 0x65,
+        Reg66H_Gpio1_Volt_HI8 = 0x66,
+        Reg67H_Gpio1_Volt_LO4 = 0x67,
+        Reg68H_Gpio2_Volt_HI8 = 0x68,
+        Reg69H_Gpio2_Volt_LO4 = 0x69,
+        Reg6AH_Gpio3_Volt_HI8 = 0x6A,
+        Reg6BH_Gpio3_Volt_LO4 = 0x6B,
+        Reg70H_Batt_Power_HI8 = 0x70,
+        Reg71H_Batt_Power_MI8 = 0x71,
+        Reg72H_Batt_Power_LO8 = 0x72,
+        Reg78H_Batt_Volt_HI8 = 0x78,
+        Reg79H_Batt_Volt_LO4 = 0x79,
+        Reg7AH_Batt_Chrg_Curr_HI8 = 0x7A,
+        Reg7BH_Batt_Chrg_Curr_LO5 = 0x7B,
+        Reg7CH_Batt_DisChg_Curr_HI8 = 0x7C,
+        Reg7DH_Batt_DisChg_Curr_LO5 = 0x7D,
+        Reg7EH_Aps_Voltage_HI8 = 0x7E,
+        Reg7FH_Aps_Voltage_LO4 = 0x7F,
+
+        // Group 5 - Battery Coulomb Counter
+        RegB0H_Batt_Chrg_Coulomb3 = 0xB0,
+        RegB1H_Batt_Chrg_Coulomb2 = 0xB1,
+        RegB2H_Batt_Chrg_Coulomb1 = 0xB2,
+        RegB3H_Batt_Chrg_Coulomb0 = 0xB3,
+        RegB4H_Batt_DisChg_Coulomb3 = 0xB4,
+        RegB5H_Batt_DisChg_Coulomb2 = 0xB5,
+        RegB6H_Batt_DisChg_Coulomb1 = 0xB6,
+        RegB7H_Batt_DisChg_Coulomb0 = 0xB7,
+        RegB8H_Coulomb_Counter_Ctrl = 0xB8
+    };
+
+    struct AxpInitRegType
+    {
+        Register reg;
+        uint8_t data;
+    };
+
+    using axp_init_reg_t = AxpInitRegType;
```

Comment:
Aliasing the struct seems unnecessary to me since it is just a name change, i.e. it doesn't reduce clutter. How about using `AxpInitReg` throughout the class?

Reply:
OK